src/stores/tutorial-store.js

```diff
@@ -0,0 +1,87 @@
+import { observable, action, computed } from 'mobx';
+
+export default class TutorialStore {
+    @observable showAgain = true;
+    @observable modalIsOpen = true;
+    @observable strategy = 'martingale';
+    @observable tradeOptions = {
+        market  : 'm1',
+        contract: 'c1',
+        stake   : 0,
+        size    : 0,
+        loss    : 0,
+        profit  : 0,
+    };
+
+    constructor() {
+        this.showAgain = localStorage.getItem('StarterHint') === 'true' || true;
+        this.modalIsOpen = this.showAgain;
+    }
+
+    changeStrategy = async () => {
```

Comment:
IMO it's better to rename this to `updateStrategy` as it more clearly describes what's happening.